packages/webpack/mixins/optimizations/mixin.core.js

```diff
@@ -1,5 +1,6 @@
 const minimatch = require('minimatch');
 const { Mixin } = require('hops-mixin');
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
```

Comment:
This has been suggested by @ZauberNerd in order to group changes made to the Babel setup. ([comment](https://github.com/xing/hops/pull/1408#discussion_r523965531))

Reply:
Hmm, sounds like it's against the current architecture of hops, but that's up to you :smiley:

Reply:
Thinking about this further - it could also make sense to place the code in `hops-react`, because it is only applicable to react code.

But I don't have a strong opinion for either solution at the moment. IMHO the webpack mixins are a mess at the moment and at some point we should restructure them properly.

Reply:
@ZauberNerd Moved it over to `hops-react`. PTAL again! 🙏